src/config/index.js

```diff
@@ -1,10 +1,10 @@
 import packageInfo from "../../package.json";
 import language from "./language.json";
 import settings from "./settings.json";
-import config from "./env.js";
+import env from "./env.js";
 
 export const taskName = packageInfo.name;
 export const taskVersion = packageInfo.version;
 export const LANGUAGE = language;
 export const SETTINGS = settings;
-export const CONFIG = config;
+export const ENV = env;
```

Comment:
Because then you just have to do 
```suggestion
export ENV ;
```